.github/workflows/test.yml

```diff
@@ -405,6 +405,12 @@ jobs:
     name: "vz"
     runs-on: macos-13
     timeout-minutes: 120
+    strategy:
+      fail-fast: false
+      matrix:
+        template:
+        - templates/experimental/vz.yaml
+        - hack/test-templates/test-vz-fedora.yaml
```

Comment:
Can we just use `templates/fedora.yaml` ?

`test-templates.sh` can be modified to receive `--vm-type=vz` via arg or env

Reply:
Didn't know we can use like this. I'll update.